packages/lesswrong/components/sunshineDashboard/SunshineNewUsersItem.tsx

```diff
@@ -88,15 +88,24 @@ const SunshineNewUsersItem = ({ user, classes, updateUser, allowContentPreview=t
 
   if (hidden) { return null }
 
+  const deletedPostCount = user.maxPostCount - user.postCount
```

Comment:
This also includes drafts, which are not deleted posts. Maybe `hiddenPostCount`?